lib/service/service.go

```diff
@@ -4685,6 +4685,12 @@ func (process *TeleportProcess) waitForAppDepend() {
 
 // registerExpectedServices sets up the instance role -> identity event mapping.
 func (process *TeleportProcess) registerExpectedServices(cfg *servicecfg.Config) {
+	// Register additional expected services for this Teleport instance.
+	// Meant for enterprise support.
+	for _, service := range cfg.AdditionalExpectedRoles {
```

Comment:
Maybe:

```suggestion
	for _, roleAndIdentity := range cfg.AdditionalExpectedRoles {
```

or some minimal abbreviation like `r`?